Rakefile

```diff
@@ -49,3 +49,39 @@ Rake::Task['db:test:load'].enhance do
   Rake::Task['multitenant:test:triggers'].invoke
   Rake::Task['db:test:procedures'].invoke
 end
+
+# In Rails 5.2 the `load_config` task was made dependent on `environment`
```

Comment:
I'm out of ideas on this one. I'm not sure there is a real solution exists. Perhaps if some day we have time to update Rails not to require `environment` for everything.

tbh I find this credentials feature as an anti-feature in its original idea to keep various credentials in the repo. It can only help small projects with a few trusted contributors.